recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
You're right. But I can't find any valid way of detecting dom changes for IE8. Maybe just use `setTimeout(..)`?